### ISSUE TITLE
inputfileFast.yaml gain values

### ISSUE THREAD

**Issue by manugr788** [IvS-KULeuven/PlatoSim3#696]
currently inputfile.yaml and inputfileFast.yaml have the same CCD gain and FEE gain. I thought (but I am not sure) that the gain will be different for F-CAM and N-CAM.
May you check with Anko the last official values for gain for fast camera and update inputfileFast.yaml accordingly.

**Comment by JorisDeRidder** [comment]
Hi! I contacted DLR and it seems that no reliable numbers for the gain difference between N-Cam and F-Cam are yet available. I did discover that the CCD gain mentioned in Anko's document (and which PlatoSim uses) is no longer up-to-date. Rather than a gain of 1.80 𝜇V/e- the measurements point to a gain of 2.14 𝜇V/e- for the E side and 2.21 𝜇V/e- for the F side of the N-Cam CCD. I'll adapt this in the PlatoSim input yaml files asap. 

I'm looking forward to the availability of the Plato Parameter Database with its Python interface so that we can auto-generate the PlatoSim inputfiles with the latest and most reliable mission parameters...
